### ISSUE TITLE
Check & Update dependencies - to react-native-video 3.x

### ISSUE THREAD

**Issue by xstable** [itsnubix/react-native-video-controls#114]
Currently I got a warning for react-native-video and video-controls dependencies.
Please check and update the dependencies for react-native-video-controls

> error "react-native-video" is wrong version: expected "3.2.0", got "3.1.0"
> error "react-native-video-controls#react-native-video@^2.0.0" doesn't satisfy found match of "react-native-video@3.1.0"
>

**Comment by xstable** [comment]
@kylemilloy does this Component still under maintenance?

**Comment by kylemilloy** [comment]
Yes...I've just been really really ridiculously busy and haven't had a video-based RN project come in to allocate some hours to this. Sorry guys...I will be updating this in the future...it is getting a little stale now.

**Comment by xstable** [comment]
@kylemilloy thank you for your response. We honor that you share this component. So no stress for that.

**Comment by FadiAboMsalam** [comment]
Any updates on this ?

**Comment by kylemilloy** [comment]
Likely when moving this package to v3 it'll cut support for RN 0.60 and RNV 4/5...at this time you can install peer deps yourself to which ever version and besides npm grumbling at you this doesn't break anything...low priority for me right now unless you wanna submit a PR.
